.github/workflows/main.yml

```
@@ -15,20 +16,66 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+          
+      - name: Check if version changed
+        id: check_version_change
+        run: |
+          # Get current version
+          CURRENT_VERSION=$(grep '^version: ' pubspec.yaml | cut -d ' ' -f 2 | tr -d '\r')
+          echo "Current version: $CURRENT_VERSION"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)